templates/arista_eos_show_ip_helper-address.template

```diff
@@ -0,0 +1,12 @@
+Value Required INTERFACE (\S+)
+Value List IP_HELPER (\d+\.\d+\.\d+\.\d+|\S+)
+
+Start
+ ^DHCP
+ ^\S -> Continue.Record
```

Comment:
Can you change L6 to `^Interface -> Continue.Record`?

Once that is changed, I will merge it.

Thanks for your work on the template!

Reply:
Uclvri kp LS